samples/golang/mywebapi/src/app/hello.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func sayHelloHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello")
+}
+
+func pingHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Ping")
+}
+
+func main() {
+	fmt.Println("Hello World!")
```

Comment:
Probably more useful to output a message after ListenAndServe has completed like `Server is listening on port 80`.

Reply:
Well, I guess it never completes, but is there some callback or something you can listen to?

Reply:
I can break it into Listen and Serve separately.